src/main/java/org/kuali/kfs/module/ar/service/impl/ContractsGrantsInvoiceCreateDocumentServiceImpl.java

```diff
@@ -1093,7 +1092,13 @@ public List<ContractsGrantsInvoiceDetail> generateValuesForCategories(
                             .add(accountObjectCodeTotalBilled.getTotalBilled()));
                 }
             }
-
+            
+            /*
+             * CU Customization back port FINP-10147
```

Comment:
To make it easier for our future KualiCo upgrades to change the sections of these methods that do not pertain to the backports, it would be helpful to also include comments that mark the ends of the customized blocks.